assets/js/partials/event-card.hbs

```diff
@@ -1,5 +1,5 @@
 <div class="grid grid-cols-1 gap-x-10 gap-y-6 sm:grid-cols-2 lg:grid-cols-4 xl:gap-x-16 w-4/5 m-auto break-words">
-  {{#each events}}
+  {{#foreach events}}
```

Comment:
Rejecting this change. The Handlebars partial here is not used directly in the Ghost theme and in fact is used in vanilla Handlebars, which uses the helpers specified in https://handlebarsjs.com/guide/builtin-helpers.html (and specifically `#each` instead of `#foreach`). See the import in https://github.com/AdvisorySG/ghost-advisory-theme/blob/500712dc5a601cae80ae350766254ade2a042fdd/assets/js/eventCardFetcher.js and Gulpfile for more details.

Reply:
Changes reverted
https://github.com/AdvisorySG/ghost-advisory-theme/blob/9955faf3f65711e2b1e7cd5c5636a2729c927720/package.json#L39-L43

Reply:
https://github.com/AdvisorySG/ghost-advisory-theme/blob/afd9df4f2356d64bf28b2bb68afdca2c5fe47c42/assets/js/partials/event-card.hbs#L2-L26

Changes reverted